diff --git a/android/src/main/java/com/bastengao/usbserialport/UsbSerialportForAndroidModule.java b/android/src/main/java/com/bastengao/usbserialport/UsbSerialportForAndroidModule.java
--- a/android/src/main/java/com/bastengao/usbserialport/UsbSerialportForAndroidModule.java
+++ b/android/src/main/java/com/bastengao/usbserialport/UsbSerialportForAndroidModule.java
@@ -18,6 +18,8 @@
 import com.facebook.react.bridge.WritableMap;
 import com.facebook.react.module.annotations.ReactModule;
 import com.facebook.react.modules.core.DeviceEventManagerModule;
+import com.hoho.android.usbserial.driver.CdcAcmSerialDriver;
+import com.hoho.android.usbserial.driver.ProbeTable;
 import com.hoho.android.usbserial.driver.UsbSerialDriver;
 import com.hoho.android.usbserial.driver.UsbSerialPort;
 import com.hoho.android.usbserial.driver.UsbSerialProber;
@@ -129,7 +131,10 @@
             return;
         }

-        UsbSerialDriver driver = UsbSerialProber.getDefaultProber().probeDevice(device);
+        ProbeTable customTable = UsbSerialProber.getDefaultProbeTable();
+        customTable.addProduct(0x303A, 0x1001, CdcAcmSerialDriver.class); // ESP32-C3/S3 USB Serial/JTAG
+        UsbSerialProber prober = new UsbSerialProber(customTable);
+        UsbSerialDriver driver = prober.probeDevice(device);
         if (driver == null) {
             promise.reject(CODE_DRIVER_NOT_FOND, "no driver for device");
             return;
@@ -153,6 +158,9 @@ public class UsbSerialportForAndroidModule extends ReactContextBaseJavaModule im
         try {
             port.open(connection);
             port.setParameters(baudRate, dataBits, stopBits, parity);
+            port.setDTR(true);
+            port.setRTS(true);
+
         } catch (IOException e) {
             try {
                  port.close();
diff --git a/android/build.gradle b/android/build.gradle
--- a/android/build.gradle
+++ b/android/build.gradle
@@ -3,7 +3,6 @@
         repositories {
             google()
             mavenCentral()
-            jcenter()
         }

         dependencies {
@@ -51,7 +50,6 @@ repositories {
     maven { url 'https://jitpack.io' }
     google()
     mavenCentral()
-    jcenter()
 }

 dependencies {
