diff --git a/android/src/main/java/com/bastengao/usbserialport/UsbSerialportForAndroidModule.java b/android/src/main/java/com/bastengao/usbserialport/UsbSerialportForAndroidModule.java
index 7e2028288172afdfb08e6ac0d4d07b69d5885902..a9180b049afd7d8e48900964365bb4d0783a8769 100644
--- a/android/src/main/java/com/bastengao/usbserialport/UsbSerialportForAndroidModule.java
+++ b/android/src/main/java/com/bastengao/usbserialport/UsbSerialportForAndroidModule.java
@@ -153,6 +153,9 @@ public class UsbSerialportForAndroidModule extends ReactContextBaseJavaModule im
         try {
             port.open(connection);
             port.setParameters(baudRate, dataBits, stopBits, parity);
+            port.setDTR(true);
+            port.setRTS(true);
+
         } catch (IOException e) {
             try {
                  port.close();
diff --git a/android/build.gradle b/android/build.gradle
--- a/android/build.gradle
+++ b/android/build.gradle
@@ -3,7 +3,6 @@
         repositories {
             google()
             mavenCentral()
-            jcenter()
         }
 
         dependencies {
@@ -51,7 +50,6 @@ repositories {
     maven { url 'https://jitpack.io' }
     google()
     mavenCentral()
-    jcenter()
 }
 
 dependencies {
diff --git a/android/src/main/AndroidManifest.xml b/android/src/main/AndroidManifest.xml
index b88623856198359153c479f17413321f18b7b9bd..57636d24e79f9b8020fe172c2256670f791a20ae 100644
--- a/android/src/main/AndroidManifest.xml
+++ b/android/src/main/AndroidManifest.xml
@@ -1,4 +1,4 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
           package="com.bastengao.usbserialport">
-    <uses-feature android:name="android.hardware.usb.host" />
+    <uses-feature android:name="android.hardware.usb.host" android:required="false" />
 </manifest>
